sh/tests/to-git-or-not-to-git_test.sh

```diff
@@ -8,6 +8,10 @@ FILENAME="student/to-git-or-not-to-git.sh"
 
 if [ -f ${FILENAME} ]; then
     if [ -s ${FILENAME} ]; then
+        submitted=$(bash student/to-git-or-not-to-git.sh)
+        expected=$(bash solutions/to-git-or-not-to-git.sh)
+        echo $submitted
+        echo $expected
```

Comment:
What is the reason for these `echo`s?

Reply:
my bad, it was for testing purposes and i forgot to remove it